basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/DBLTIService.java

```diff
@@ -174,7 +174,45 @@ public Object insertToolDao(Object newProps, String siteId, boolean isAdminRole,
 
 	public Map<String, Object> getToolDao(Long key, String siteId, boolean isAdminRole)
 	{
-		return getThingDao("lti_tools", LTIService.TOOL_MODEL, key, siteId, isAdminRole);
+		Map<String, Object> retval = getThingDao("lti_tools", LTIService.TOOL_MODEL, key, siteId, isAdminRole);
+
+		// SAK-50378 - If requested (i.e. we found something wrong after Sakai-25 is released), we fake up deprecated values
+		// This is quicker than a patch and can be used while a patch is being developed and/or for
+		// testing to see if the deleted columns caused a regression
+		// This entire block of code can be removed after Sakai 25.2 if it is not needed by then
+        String ltiFakeDeprecated = serverConfigurationService.getString("sakai-sak-50328-fake-deprecated-values", "false");
```

Comment:
maybe we can talk about this?

Reply:
Sure - I will leave it in and we can review at the Tuesday meeting.  I will fix the indentation tho :)